lib/cdo/activity_constants.rb

```diff
@@ -10,4 +10,7 @@ module ActivityConstants
   MAXIMUM_NONOPTIMAL_RESULT = 29
   FREE_PLAY_RESULT = 30
   BEST_PASS_RESULT = 100
+
+  REJECTED_RESULT = 150
```

Comment:
These constants are specific to peer review and should have that incorporated in their name